frontend/2014-kashmir-floods-explorer/index.html

```
@@ -0,0 +1,397 @@
+    <meta content="width=device-width, initial-scale=1.0" name="viewport"/>
+    <title>2014 Jammu & Kashmir Floods | Incredible India</title>
+    <meta name="description" content="Document the 2014 floods and the geography of flood risk in the Kashmir Valley">
+    
+    <!-- Preload Critical Fonts -->
+    <link rel="preload" href="../../assets/fonts/Outfit-400.woff2" as="font" type="font/woff2" crossorigin>
+    <link rel="preload" href="../../assets/fonts/Outfit-700.woff2" as="font" type="font/woff2" crossorigin>
+    <link rel="preload" href="../../assets/fonts/PlayfairDisplay-400.woff2" as="font" type="font/woff2" crossorigin>
+    
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.7.2/css/all.min.css"/>
+    <link rel="stylesheet" href="../../styles.css"/>
+    <link rel="stylesheet" "../pages-common.css"/>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)